tests/base.py

```diff
@@ -692,13 +692,15 @@ def assertRecordsEqual(self, stream, expected_record, sync_record):
                 self.assertDictEqual(expected_record, sync_record)
 
         def assertDictEqualWithOffKeys(self, expected_record, sync_record, off_keys=frozenset()):
-            self.assertParentKeysEqual(expected_record, sync_record)
```

Comment:
Why would this line be commented? It's just asserting the top-level keys are identical?

Reply:
Anything that would fail here, we would catch in the assertions in `test_all_fields.py`. Since there are new fields for `payments` now I am already implementing a workaround in the tests for it. If we wanted to keep it we'd need two of these OffKeys assertion methods; one method that checks parent keys match, and another that does not. But since it is covered elsewhere I dropped it.

Reply:
Addressed this in most recent commit.